task/fbc-validation/0.1/fbc-validation.yaml

```diff
@@ -36,7 +38,8 @@ spec:
         ### Try to extract binaries with configs > check binaries functionality > check opm validate ###
         if [ ! -f ../sanity-inspect-image/image_inspect.json ] || [ -z $(cat ../sanity-inspect-image/image_inspect.json) ]; then
           echo "File ../sanity-inspect-image/image_inspect.json is not generated correctly, please check HACBS_TEST_OUTPUT of task sanity-inspect-image"
-          HACBS_TEST_OUTPUT="$(make_result_json -r ERROR -t 'File ../sanity-inspect-image/image_inspect.json is not generated correctly, please check HACBS_TEST_OUTPUT of task sanity-inspect-image!')"
+          note="Task $(context.task.name) meets error: File ../sanity-inspect-image/image_inspect.json is not generated correctly, please check HACBS_TEST_OUTPUT of task sanity-inspect-image!"
```

Comment:
SR: note="Task $(context.task.name) meets error: File `../sanity-inspect-image/image_inspect.json` did not generate correctly. Check `HACBS_TEST_OUTPUT` in the `sanity-inspect-image` task."

1. IBM Style Guide [absolutely forbids](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#s) the use of the phrase "sanity check" and the word "sanity" in any Red Hat documentation as potentially offensive to neurodiverse people. We have to change these here and globally.
2. What is `HACBS_TEST_OUTPUT`? A component? An argument? A file?
3. Where can a user find this task?
4. What are they checking for?

Reply:
1, we may need to use another name, but we have use this word `sanity` in a lot of places and need to update them together.
2. `HACBS_TEST_OUTPUT` is the exposed result of tekton task and can be visited by other tasks.
3. It can be found in the same pipeline with task fbc-validation. Updated note to "Task $(context.task.name) failed: File /workspace/hacbs/sanity-inspect-image/image_inspect.json did not generate correctly. For detailed error, check task result `HACBS_TEST_OUTPUT` in the task `sanity-inspect-image` task of the pipeline."
4.  If they see this inspect error, they can check the error in task `sanity-inspect-image`

Reply:
Story [HACBS-1945](https://issues.redhat.com/browse/HACBS-1945) is filed to rename the task name.